app/models/miq_action.rb

```diff
@@ -353,27 +353,9 @@ def action_evm_event(action, rec, inputs)
       :timestamp  => Time.now.utc
     }
     opts = default_options.merge(action.options || {})
+    opts[:target] = rec
 
-    opts[:ems_id] = rec.ext_management_system.id if rec.respond_to?(:ext_management_system) && rec.ext_management_system
-    case rec
-    when VmOrTemplate
-      opts[:vm_or_template_id] = rec.id
-      opts[:vm_name]           = rec.name
-      opts[:vm_location]       = rec.location
-      unless rec.host.nil?
-        opts[:host_id]    = rec.host.id
-        opts[:host_name]  = rec.host.name
-      end
-    when Host
-      opts[:host_id]    = rec.id
-      opts[:host_name]  = rec.name
-    when EmsCluster
-      opts[:ems_cluster_id]    = rec.id
-      opts[:ems_cluster_name]  = rec.name
-      opts[:ems_cluster_uid]   = rec.uid_ems
-    end
```

Comment:
@lfu @gtanzillo Should we retain setting the existing columns as well as adding in the target?  I am thinking that we are filtering on at least `ems_id` in a number of places, like timelines.